content/ai/concepts/chatgpt/chatgpt.md

```diff
@@ -0,0 +1,103 @@
+
+### Recurrent Neural Networks
+
+One limitation of ANNs is that they don’t have the concept of time sequences and the input size is fixed. So for language processing where you have a sequence of words or a sentence, Recurrent Neural Networks (RNNs) were created.
+
+RNNs feed decision results back in the system. So for example if we wanted to translate the sentence “What time is it?” by the time the word “it” is encountered, the system would already know all about the question.
+
+While RNNs have been very useful in doing language translation, they do not scale well with large amounts of information. Also computation can not be done in parallel.
+
+### Breakthrough with Transformers
+
+The breakthrough that allowed BERT and ChatGPT-based models is called `Transformers`. This was introduced in a paper from 2019 entitled `“Attention is all you need”`. It was presented as a way to build machine translation but to have much faster parallel execution. It also has a new concept called `Self Attention` to further make sense of long language sequences. It creates connections between different parts of the sequence.
+
+So how does it all really work? In the `Transformer` based model each word is transformed into a token. As tokens are read as part of an input sentence the model starts predicting what the output should be - similar in concept to sentence completion.
+
+Due to the use of `Transformers` in creating the model, the potential output is not just a prediction of the next word. Because the model is trained with deep understanding and `Attention` whole sentences and groupings of text are generated. The model used by ChatGPT can accept several thousand tokens as input .
+
+## Risks and Limitations
+
+### Misinformation
+
+Because current LLMs are trained against data sets from some period of time in the past they don't always give accurate information. Also their correctness is only as good as the data they are trained with. This doesn't always result in accurate results.
+
+### Disinformation
+
+ChatGPT and LLMs in general often give confident and plausible generated results but they are wrong or do not make sense. In AI these are called Hallucinations. With prompt engineering, ChatGPT can be told to not give an answer unless it has high confidence in the results. This can reduce hallucinations but not completely solve the problem.
+
+### Data Security
+
+Using ChatGPT requires the sending of information over the internet. Sending personal or sensitive information presents a possible security risk.
+
+### Bias
+
+The internet is full of biased negative information. For ChatGPT to be useful this information needs to be filtered so it is not produced in responses. OpenAI has identified this as a major concern with ChatGPT and LLMs in general.
```

Comment:
```suggestion
The internet is full of biased negative information. For ChatGPT to be useful, this information needs to be filtered so that it is not produced in responses. OpenAI has identified this as a major concern with ChatGPT and LLMs in general.
```

Reply:
Resolved issues